database/migrations/2018_04_16_162649_basic_tables.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class BasicTables extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('workflows', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger("authorId");
+            $table->timestamp("created")->useCurrent();
+            $table->timestamp("updated")->nullable();
+            $table->string("languageCode",2);
```

Comment:
You might want a separate table for this and reference it using an id

Reply:
Thanks for the tip. Languages are not our current priority so we will decide on it later.